### PR TITLE
docs: fix broken NgZone code example

### PR DESCRIPTION
**Pull request by juristr**
The current code example was broken as there were a couple of syntax errors. This commit fixes the demo.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

The current inline code example that contains a demo for NgZone was broken, containing a couple of syntax errors. Found it when copy & pasting it from the angular.io docs.

Issue Number: N/A

## What is the new behavior?

This commit fixes the demo example. Tried it here: https://stackblitz.com/edit/angular-ngzones-demo

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by juristr** [comment]
@wardbell sorry for poking you directly. Could you or someone give a quick look at this. It's an easy fix of a code example which is wrong on the documentation site.

The build fails, but doesn't seem related to my change. Is there anything I need to update or rebase to make it pass?

thx :)

File: packages/core/src/zone/ng_zone.ts [hunk 1]
```diff
@@ -63,17 +63,18 @@ import {EventEmitter} from '../event_emitter';
  *     this.progress = 0;
  *     this._ngZone.runOutsideAngular(() => {
  *       this._increaseProgress(() => {
- *       // reenter the Angular zone and display done
- *       this._ngZone.run(() => {console.log('Outside Done!') });
- *     }}));
+ *         // reenter the Angular zone and display done
+ *         this._ngZone.run(() => {console.log('Outside Done!') });
```
Comment (gkalpak) [inline packages/core/src/zone/ng_zone.ts]: While you're at it, would you like to also fix:

`{console.log('Ouside Done!') }` --> `{ console.log('Ouside Done!'); }`

**Comment by juristr** [comment]
@mary-poppins Oh cool, didn't know, very helpful. Just verified and my corrected example runs properly :)

**Review by gkalpak — CHANGES_REQUESTED** [review]
Ideally, this should not be hardcoded, but come from an actual running [example app][1] (just in case you feel adventurous) :smiley:

Can you please rebase on latest master and change the commit message to `docs(core): ...`?

Other than that LGTM (as soon as Travis is happy).

[1]: https://github.com/angular/angular/tree/master/packages/examples/core

File: packages/core/src/zone/ng_zone.ts (hunk 1 above)
Comment (juristr) [reply packages/core/src/zone/ng_zone.ts]: Oh, right, sure 😄, but without introducing again the "Ouside.." typo 😉. Will do immediately

**Comment by juristr** [comment]
> Ideally, this should not be hardcoded, but come from an actual running example app (just in case you feel adventurous) 😃

Aha, I see what you mean. Ya for sure. I'll give it a look whether I can convert it to a runnable example. I'll ping you back in case I'm stuck 😄

**Comment by juristr** [comment]
@gkalpak Ok, rebased and updated my code. Checks seem to pass now properly. For now this should work and be ready to merge. I'll check the conversion into a runnable example later. Will come back with another PR, promised :)

File: packages/core/src/zone/ng_zone.ts [hunk 2]
```diff
@@ -63,17 +63,18 @@ import {EventEmitter} from '../event_emitter';
  *     this.progress = 0;
  *     this._ngZone.runOutsideAngular(() => {
  *       this._increaseProgress(() => {
- *       // reenter the Angular zone and display done
- *       this._ngZone.run(() => {console.log('Outside Done!') });
- *     }}));
+ *         // reenter the Angular zone and display done
+ *         this._ngZone.run(() => {console.log('Outside Done!'); });
```
Comment (gkalpak) [inline packages/core/src/zone/ng_zone.ts]: Nit there could be a space before `console` :grin:

**Review by gkalpak — APPROVED** [review]
LGTM :+1: (but still needs approval from `core` owners :grin:)

**Comment by gkalpak** [comment]
@juristr, I had hoped for an observable of PRs, but a promise will do for now :stuck_out_tongue:

File: packages/core/src/zone/ng_zone.ts (hunk 2 above)
Comment (juristr) [reply packages/core/src/zone/ng_zone.ts]: 🙄 😉 
If ur ok with it, I'd leave it like this and fix it when I convert it to an executable example, as discussed earlier.

Comment (gkalpak) [reply packages/core/src/zone/ng_zone.ts]: Oh, look, it's fixed :wink:

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**